tee-worker/litentry/core/credentials/src/jsonschema.rs

```diff
@@ -0,0 +1,123 @@
+// Copyright 2020-2023 Trust Computing GmbH.
+// This file is part of Litentry.
+//
+// Litentry is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Litentry is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Litentry. If not, see <https://www.gnu.org/licenses/>.
+
+use crate::achainable;
+use litentry_primitives::{AchainableParams, Assertion};
+use std::string::String;
+
+const BASE_URL: &str = "https://raw.githubusercontent.com/litentry/vc-jsonschema/main/dist/schemas";
+const NOT_IMPLEMENTED: &str =
```

Comment:
I'm thinking maybe something like `FALLBACK_SCHEMA` would be better? 🤔

Reply:
I'm ok with that. I wanted to signal something that needs to be done and shouldn't be optional. 

Do you agree that any new VC should have a defined schema? Like, I don't want to leave the door open to situations where we put off the schema definition for the sake of delivering and later on forgetting about it or coming up with a complex schema as it wasn't thought before.